content/about/_index.md

```diff
@@ -1,8 +1,12 @@
 ---
 title: "VISUALIZATION AND ANALYSIS SOFTWARE AND TECHNOLOGY"
 date: 2022-04-27T00:00:00-06:00
-mission: "NCAR’s Visualization and Analysis Software and Technology (VAST) section helps scientists, students, policy makers, and the public better understand enormous volumes of geoscientific data. Numerical simulations of complex natural phenomena, running on some of the world’s most powerful supercomputers, and a wide range of earth observing instruments all produce tremendous amounts of numerical data about the natural world. Helping explore, explain, understand, and gain insight into these complex troves of data is the mission of VAST."
 image: images/banners/limb.jpg
+#mission:
```

Comment:
I'm not sure you can leave the field blank. I think you need to delete this line altogether. 

```suggestion
```